### DISCUSSION TITLE
Vectorize support?

### DISCUSSION THREAD

**Discussion by thipperz** [cloudflare/next-on-pages#721] [General]
Is it currently possible to query a Vectorize index from the Next Pages project? Would there be a performance gain when doing so, instead of calling the vectorize worker API?

**Comment by dario-piotrowicz** [comment]
Hi @thipperz 👋 

Yes, you should be able to use Vectorize without issues using next-on-pages 🙂

But I don't think there would be any performance gain compared to using the Vectorize worker API
